pages/admin/articles.js

```diff
@@ -65,16 +65,16 @@ const ARTICLE_COLS = [
   {
     id: 'subtitle',
     className: styles['wir-table__font--size--small'],
-    formatter: text => `${text.slice(0, 100)}...`,
+    formatter: (text) => `${text.slice(0, 100)}...`,
```

Comment:
hmm, but I disabled such format: https://github.com/babajka/babajka-frontend/blob/develop/.prettierrc.js#L5

Reply:
For some reason prettier changed it's mind again and now it's ok🤷‍♂️

Reply:
I think you forget to run `npm i` after rebase, so old prettier version was used

Reply:
that is possible